toolkit/tools/internal/resources/assets/grub2/grub

```diff
@@ -17,6 +17,6 @@ GRUB_CMDLINE_LINUX_DEFAULT="{{.ExtraCommandLine}} $kernelopts"
 # =============================notice===============================
 for x in /etc/default/grub.d/*.cfg ; do
     if [ -e "${x}" ]; then
-        . "${x}"
+        . "${x}" || grub_warn "Received non-zero exit code from ${x}."
```

Comment:
do we really want to ignore errors here? I mean technically this should never happen, these *.cfg files should always only contain env var settings, so if there is anything else that actually fails it probably should let the error flow up, no?

With that said I don't have any particular problem with this since, as mentioned, the *.cfg files shouldn't ever contain anything other than setting env vars. I'll leave it up to you.

Reply:
This is the edge case where we have kernel-mshv installed but no mshv. Normally we would enforce this via the package manager (kernel-mshv Requires: mshv...). mshv is a proprietary package of close-source hypervisor binaries however; we can't have this package depend on it directly.  

Instead, I have the 50_mariner_mshv.cfg determine whether or not the mshv components are installed at mkconfig-time and either set GRUB_DEFAULT=dom0 or return 1.